external/webserver/http_client.c

```diff
@@ -1242,10 +1245,17 @@ int http_send_response(struct http_client_t *client, int status, const char *bod
 {
 	const char* status_message = (status == 200) ? "OK" : body;
 
-	return http_send_response_helper(client, status, status_message, body, headers);
+	int body_len = strlen(body);
```

Comment:
Why do you use strlen for it???

Reply:
http_send_response() http_send_response_with_status_msg() API arguments can not be modified as it is being used by other applications.

Reply:
We have introduced new API http_send_response_with_status() which does not use strlen to calculate the length of the body.

Reply:
I believe the question here is that since "body" is a constant character array anyways,
Why not pass the parameter to `http_send_response_helper` as "strlen(body)" instead of using an extra variable here?

Reply:
That was not the question which you are asking for it....
Point is to remove the strlen function that is difficult here.

Reply:
Why is it difficult????
How about adding input parameter body_len????